src/Promitor.Core.Scraping/Configuration/Serialization/Deserializer.cs

```diff
@@ -17,9 +24,39 @@ protected Deserializer(ILogger logger)
             Logger = logger;
         }
 
-        public abstract TObject Deserialize(YamlMappingNode node);
+        public virtual TObject Deserialize(YamlMappingNode node, IErrorReporter errorReporter)
```

Comment:
For the final PR make sure to add XML docs to public methods please

Reply:
Sure. Have updated.